app/AndroidManifest.xml

```diff
@@ -28,7 +28,7 @@
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
-
+    <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM" />
```

Comment:
This permission can be revoked by the user or system at any time and we need to check if the permission is granted before using it otherwise direct user to grant this permission. [reference](https://developer.android.com/training/scheduling/alarms#using-schedule-exact-permission)